python/setup.py

```diff
@@ -342,14 +414,29 @@ def build(build_python, build_java, build_cpp):
         logger.warning("Expected Bazel version {} but found {}".format(
             ".".join(map(str, SUPPORTED_BAZEL)), bazel_version_str))
 
+    root_dir = os.path.join(
+        os.path.abspath(os.environ["SRC_DIR"]), "..", "bazel-root")
+    out_dir = os.path.join(
+        os.path.abspath(os.environ["SRC_DIR"]), "..", "bazel-out")
+
+    for d in (root_dir, out_dir):
+        if not os.path.exists(d):
+            os.makedirs(d)
+
+    bazel_options = [
+        "--output_user_root=" + root_dir, "--output_base=" + out_dir, "build",
```

Comment:
and I think these should also be triggered by the same condition, like `if doing_conda_build` or something